src/modules/cache-client/cache-client.service.ts

```diff
@@ -63,44 +64,39 @@ export class CacheClient implements ICacheClient {
    * After authentication runs previously failed requests
    */
   async authenticate() {
+    let tokens: AuthTokens | undefined = undefined;
+
+    // First try to refresh access token
     try {
       const refreshedTokens = await this.refreshToken();
 
-      if (refreshedTokens) {
-        getLogger().debug('[CACHE CLIENT] Setting authorization tokens');
-        if (!this.isBrowser) {
```

Comment:
Don't you think it will lead to failed authentication in line 73 in Node.js?

Reply:
I've tested it and everything is fine

Reply:
Ok. Lets try it